contracts/interfaces/IRamm.sol

```diff
@@ -83,4 +83,21 @@ interface IRamm {
   function removeBudget() external;
 
   /* ========== EVENTS ========== */
+
+  // Input
+  error OneInputOnly();
+  error OneInputRequired();
+
+  // Expiry
+  error SwapExpired(uint deadline, uint blockTimestamp);
+
+  // Minimum amount out
+  error NxmOutLessThanMinAmountOut(uint nxmOut, uint minTokensOut);
+  error EthOutLessThanMinAmountOut(uint ethOut, uint minTokensOut);
```

Comment:
Can you please merge these two in a single custom error? `InsufficientOutputAmount` or something similar